### PR TITLE
Added basic support for the Symbol type introduced in ECMAScript 6.

### PR DESCRIPTION
**Pull request by noel-smith**
This prevents segmentation faults when evaluating a statement that results in a Symbol.

Related issue: https://github.com/sqreen/PyMiniRacer/issues/82

### CODE REVIEW

File: tests/test_eval.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import six
 
 from py_mini_racer import py_mini_racer
+from py_mini_racer.py_mini_racer import JSSymbol
```
Comment (Taiki-San) [inline tests/test_eval.py]:
```suggestion
```

File: tests/test_eval.py [hunk 2]
```diff
@@ -97,6 +98,11 @@ def test_max_memory(self):
                 }''', max_memory=200000000)
 
 
+    def test_symbol(self):
+        res = self.mr.eval('Symbol.toPrimitive')
+        self.assertEqual(type(res), JSSymbol)
```
Comment (Taiki-San) [inline tests/test_eval.py]:
```suggestion
        self.assertEqual(type(res), py_mini_racer.JSSymbol)
```

**Comment by noel-smith** [comment]
Hey, no problem - happy to help. I've updated with the changes you suggested.

**Comment by Taiki-San** [comment]
Sorry for the time it took, thanks a lot for your contribution!

**Comment by noel-smith** [comment]
No problem - thanks for merging!

**Pull request merged**